packages/pn-commons/src/components/CodeModal/CodeInput.tsx

```diff
@@ -116,6 +117,17 @@ const CodeInput = ({ initialValues, isReadOnly, hasError, onChange }: Props) =>
     }
   };
 
+  const pasteHandler = (event: ClipboardEvent<HTMLDivElement>, index: number) => {
+    const copiedCode = event.clipboardData.getData('text');
+    if (Number(copiedCode)) {
```

Comment:
We can have a copiedCode that has less or more digit than requested. How do we handle this cases?